conf/mccleary.config

```diff
@@ -0,0 +1,118 @@
+params {
+  config_profile_description = 'Yale YCRC Configurations'
+  config_profile_contact = 'Ryan A. Hagenson'
+}
+
+profiles {
+  mccleary {
+    executor {
+      submitRateLimit = '200/1h'
+    }
+
+    singularity {
+      enabled = true
+    }
+
+    params {
+      igenomes_base = '/gpfs/gibbs/data/genomes/'
```

Comment:
Awesome you added this!

Reply:
oh yes I was involved in the discussions, that's not happening any time soon :)
but yes, we should check that the data is regularly updated, I will try to figure out how this is done

Reply:
hmm, I'm unsure that this will work, as it does not seem to be an exact copy of the nf-core AWS igenomes base, so it's missing some of the files that we have there. Do you happen to know who handles the reference data there?

Reply:
I do not. The [documentation](https://docs.ycrc.yale.edu/clusters/mccleary/#public-datasets) just states a general contact for any updates/requests.

Reply:
ok, then we should remove this parameter until we have somewhere an exact copy of the nf-core igenomes reference data, as this one won't work well